src/main/features/core/lastFM.js

```diff
@@ -141,11 +143,14 @@ Emitter.on('lastfm:auth', () => {
 let currentRating = {};
 Emitter.on('change:track', (event, details) => {
   currentRating = {};
-  updateNowPlaying(details.title, details.artist, details.album);
+  // Last.fm isn't accepting 'Unknown Album'
+  updateNowPlaying(details.title, details.artist, (details.album === 'Unknown Album' ? undefined : details.album), Math.round(details.duration / 1000));
 });
 
 Emitter.on('change:track:scrobble', (event, details) => {
-  updateScrobble(details.title, details.artist, details.album, details.timestamp);
+  if (details.duration > 30 * 1000) { // Scrobble only tracks longer than 30 seconds
+    updateScrobble(details.title, details.artist, (details.album === 'Unknown Album' ? undefined : details.album), details.timestamp, Math.round(details.duration / 1000));
```

Comment:
Actually, can we reduce the line length here and on line 147 by extracting this ternary operator out into its own line and saving the result for both calls?